docs/_includes/sections.adoc

```diff
@@ -170,13 +171,18 @@ If `sectnums` is set on the command line (or API), that overrides the value set
 
 If `sectnums!` is set on the command line (or API), then the numbers are disabled regardless of toggling within the document.
 
-////
-You can also restrict section numbering with the +sectnumlevels+ attribute.
+=== Numbering Depth
```

Comment:
We should use sentence capitalization for titles (though, admittedly, it's not consistent atm).